### PR TITLE
Add note on how to share the badge for pylint

### LINKED ISSUE #5819: Missing badge for pylint
### Current problem

Our project is using pylint, and we want a badge like in case of black to show in our README

### Desired solution

- Create a shareable badge which can be easily attached in project README like in case of [black](https://github.com/psf/black#show-your-style)

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by kurianbenoy-aot**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|   | :bug: Bug fix          |
|   | :sparkles: New feature |
|   | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

- Add note on how to share badge for pylint

Closes #5819

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -149,6 +149,23 @@ Do not forget to clone astroid_ and install the last version::
     cd astroid
     python3 -m pip install -e .
 
+Share your code style
+-------
```
Comment (DanielNoord) [inline README.rst:153]:
```suggestion
Show your usage
-----------------
```

Linting is not really code style, although `Show your usage` isn't perfect either.

File: README.rst [hunk 2]
```diff
@@ -149,6 +149,23 @@ Do not forget to clone astroid_ and install the last version::
...
+
+Place this badge at the top of your repository to let others know your project uses pylint.
```
Comment (DanielNoord) [inline README.rst:155]:
```suggestion
You can place this badge in your README to let others know your project uses pylint.
```

File: README.rst [hunk 3]
```diff
@@ -149,6 +149,23 @@ Do not forget to clone astroid_ and install the last version::
...
+
+    .. image:: https://img.shields.io/badge/linting-pylint-green
+        :target: https://github.com/PyCQA/pylint
+
+Use the badge in your project's README.md::
```
Comment (DanielNoord) [inline README.rst:160]:
```suggestion
Use the badge in your project's README.md (or any other Markdown file)::
```

File: README.rst [hunk 4]
```diff
@@ -149,6 +149,23 @@ Do not forget to clone astroid_ and install the last version::
...
+
+    ![linting: pylint](https://img.shields.io/badge/linting-pylint-green)](https://github.com/PyCQA/pylint)
+
+Use the badge in your projects README.rst::
```
Comment (DanielNoord) [inline README.rst:164]:
```suggestion
Use the badge in your projects README.rst (or any other rst file)::
```

**Review by DanielNoord — APPROVED** [review]
Minor changes. Thanks @kurianbenoy-aot!

File: README.rst [hunk 5]
```diff
@@ -149,6 +149,23 @@ Do not forget to clone astroid_ and install the last version::
     cd astroid
     python3 -m pip install -e .
 
+Share your code style
+-------
+
+Place this badge at the top of your repository to let others know your project uses pylint.
+
+    .. image:: https://img.shields.io/badge/linting-pylint-green
```
Comment (Pierre-Sassoulas) [inline README.rst:157]: The badge I made is just a proposition, if you have something better in mind it's possible to change the url to get another badge automatically.

**Review by jacobtylerwalls — CHANGES_REQUESTED** [review]
Could we use the built-in GitHub badges here?

https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

EDIT: of course, not everyone is on GitHub, but for those that are, this is built in.

File: README.rst (hunk 5 above)
Comment (kurianbenoy-aot) [reply README.rst:157]: [Imports: isort](https://pycqa.github.io/isort/)

I really liked the isort github badge. So maybe something like a mix of pylint colours - Blue and yellow would be great like below one:

linting: pylint](https://github.com/PyCQA/pylint)

Do share your thoughts @Pierre-Sassoulas ?

Comment (Pierre-Sassoulas) [reply README.rst:157]: >  maybe something like a mix of pylint logo colours

Great idea, but the blue and yellow of the pylint logo are the same than python, we might as well use something else as it's not distinguishing at all in the python ecosystem.

I like  https://img.shields.io/badge/linting-pylint-yellowgreen, it's less flashy than green or yellow and look reasonably good in light/dark theme.

File: README.rst [hunk 6]
```diff
@@ -149,6 +149,23 @@ Do not forget to clone astroid_ and install the last version::
...
+
+Use the badge in your project's README.md (or any other Markdown file)::
+
+    ![linting: pylint](https://img.shields.io/badge/linting-pylint-green)](https://github.com/PyCQA/pylint)
+
+Use the badge in your projects README.rst (or any other rst file)::
+
+    .. image:: https://img.shields.io/badge/linting-pylint-green
+        :target: https://github.com/PyCQA/pylint
+
```
Comment (jacobtylerwalls) [inline README.rst:169]: No worries if my suggestion is a little _out-of-scope_ 🌨️ . I was just thinking of something like this, which works for folks who have a workflow beginning like:

```yaml
name: pylint
```
```suggestion

If you use GitHub Actions, and one of your CI workflows begins with "name: pylint", you
can use GitHub's
[workflow status badges](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-workflow-file-name)
to show an up-to-date indication of whether pushes to your default branch pass pylint.
```

File: README.rst (hunk 5 above)
Comment (kurianbenoy-aot) [reply README.rst:157]: Let's go with yellowgreenthen 👍

**Review by Pierre-Sassoulas — APPROVED** [review]
Thank you @kurianbenoy-aot and @jacobtylerwalls, great addition to the readme !

**Pull request merged**